Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -539,13 +539,15 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
         BackgroundColor,
         Plugin,
         Model,
+        HostedModel,
         Host,
     };
 
     // Pass an invalid color to remove the contents layer.
     virtual void setContentsToSolidColor(const Color&) { }
     virtual void setContentsToPlatformLayer(PlatformLayer*, ContentsLayerPurpose) { }
     virtual void setContentsToPlatformLayerHost(LayerHostingContextIdentifier) { }
+    virtual void setContentsToPlatformLayerCustom(LayerHostingContextIdentifier, ContentsLayerPurpose) { }
```

Comment:
Technically `setContentsToPlatformLayer` maps to the normal "Custom" PlatformCALayerRemoteCustom. This is like "setContentsToRemotePlatformContext" or something?

Reply:
Good one! I'll make it `setContentsToRemotePlatformContext`